automatic/logfusion/logfusion.nuspec

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>logfusion</id>
+    <version>0.0</version>
+    <title>LogFusion</title>
+    <authors>Binary Fortress Software</authors>
+    <projectUrl>http://www.logfusion.ca</projectUrl>
+    <licenseUrl>https://www.logfusion.ca/License/</licenseUrl>
+    <iconUrl>https://cdn.rawgit.com/chocolatey/chocolatey-coreteampackages/7457c4aec7a50384fc3e8787fb59e4be48b84738/icons/LogFusion.png</iconUrl>
+    <docsUrl>https://www.logfusion.ca/Help/</docsUrl>
+    <mailingListUrl>https://www.logfusion.ca/Discussions/</mailingListUrl>
+    <requireLicenseAcceptance>false</requireLicenseAcceptance>
+    <description>
+    ## Features
```

Comment:
Should contain a few lines of the software description before the Features header